### PR TITLE
Score computation v1

### PR DESCRIPTION
**Pull request by Mandloi2309**
Logic for computing score, right now it just prints the score in the console.

If approved we can add it as the validation report.

### CODE REVIEW

File: src/services/validator.service.ts [hunk 1]
```diff
@@ -217,6 +221,89 @@ class ValidatorService {
...
+      return modelScores;
+
+    }
+    
+    // Build weights dictionary
+    const weightsDictionary: { [schemaName: string]: Array<{ fieldName: string; fieldWeight: number; weightForModelType: string }> } = {};
+    validationDictionary.schemas.forEach(schema => {
+      schema.fields.forEach(field => {
+        const { meta } = field;
+        if ( meta && 'field_weight' in meta && 'weight_for_model_type' in meta && typeof meta.field_weight === 'number' && meta?.field_weight > 0 && typeof meta?.weight_for_model_type === 'string') {
```
Comment (mauroz77) [inline src/services/validator.service.ts:238]: Can we move this validation to a helper method so the conditional is not that long? Something like `if (scoreMetadataExists(meta))` or something similar?

**Review by mauroz77 — COMMENTED** [review]
Left one comment. Besides that, it looks good to me!

File: src/services/validator.service.ts (hunk 1 above)
Comment (Mandloi2309) [reply src/services/validator.service.ts:238]: Yes definitely

**Comment by Mandloi2309** [comment]
please have a look @mauroz77

**Review by mauroz77 — APPROVED** [review]

**Pull request merged**